src/base64Decode.js

```diff
@@ -0,0 +1,61 @@
+#if WASM2JS && 0 // TODO: Figure out a way to enable this kind of sharing.
+
+// Binaryen defines the following function if Wasm2JS is being used:
+//   function base64DecodeToExistingUint8Array(uint8Array, offset, b64);
+// so should reuse that when available. However that lives inside the asm module
+// for the time being, so cannot access it directly from here. Hence this block
+// is disabled atm.
+
+function base64Decode(b64) {
+#if ASSERTIONS
+  assert(b64.length % 4 == 0);
+#endif
+  return base64DecodeToExistingUint8Array(new Uint8Array(b64.length*3>>2), 0, b64);
+}
+
+#else
+
+// Precreate a reverse lookup table from chars "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/" back to bytes to make decoding fast.
+for (var base64ReverseLookup = new Uint8Array(123/*'z'+1*/), i = 25; i >= 0; --i) {
```

Comment:
This seems to break the minifier for me. I wasn't able to find a small repro, but it generates this:

```
for(var Y=new Uint8Array(123),Z=25;ba>=0;--ba)
{Y[48+ba]=52+ba;Y[65+ba]=ba;Y[97+ba]=26+ba}Y[43]=62;Y[47]=63;
```

Reply:
This looks familiar, do you still see this on tip of tree? I think/hope we've fixed it.

Reply:
If I'm right the fix was in #10522

Reply:
I'm not sure, I used `emsdk install latest` last night, does that pull ToT?

Reply:
Ah, try `install tot` then, hopefully that fixes it.

Sorry, been a while since we tagged, there were problems...